src/main/scala/mesosphere/marathon/core/task/tracker/TaskTrackerModule.scala

```diff
@@ -0,0 +1,34 @@
+package mesosphere.marathon.core.task.tracker
+
+import akka.actor.ActorRef
+import akka.event.EventStream
+import mesosphere.marathon.MarathonSchedulerDriverHolder
+import mesosphere.marathon.core.leadership.LeadershipModule
+import mesosphere.marathon.core.task.bus.TaskStatusObservables
+import mesosphere.marathon.core.task.tracker.impl.{ TaskStatusUpdateActor, KillOverdueStagedTasksActor }
+import mesosphere.marathon.health.HealthCheckManager
+import mesosphere.marathon.tasks.{ TaskIdUtil, TaskTracker }
+
+class TaskTrackerModule(leadershipModule: LeadershipModule) {
```

Comment:
Please add some documentation for this module as well the provided objects/functionality.